engine/src/test/scala/cromwell/WorkflowOutputsSpec.scala

```diff
@@ -63,7 +63,7 @@ class WorkflowOutputsSpec extends CromwellTestkitSpec {
     "Allow explicitly named inputs in the output section" ignore {
```

Comment:
Pretty sure it's there because it's in the WDL spec. I'd wager fairly confidently that nobody's using it currently